### ISSUE TITLE
ETH-Precision Don't Send Fee if no Fee When Saving Moderator Info

### ISSUE THREAD

**Issue by jjeffryes** [OpenBazaar/openbazaar-desktop#1839]
This is happening on Dispute payout precision #1838 and ethereum-master latest as of 11-27-19.

When saving a moderator fee, if it's percentage the server kicks back this error `invalid profile: percentage moderator fee should not include a fixed fee or should use (FIXED_PLUS_PERCENTAGE) feeType`

I'm not sure if this is a server or a desktop issue. It seems like it would be relatively simple to remove the fixedFee object if the type is percentage when saving?

Below is the data being sent.

```
{
  "moderator": true,
  "moderatorInfo": {
    "fee": {
      "feeType": "PERCENTAGE",
      "fixedFee": {
        "bigAmount": "0",
        "amountCurrency": {
          "code": "USD",
          "divisibility": 2
        }
      },
      "percentage": 50
    },
    "description": "This is just a test account, do not use.",
    "termsAndConditions": "No disputes will be moderated. This is only a test account.",
    "languages": [
      "en-US"
    ]
  }
}
```

**Comment by rmisio** [comment]
@jjeffryes This is now fixed on the client via c2e0dd97f10d3f4bce69b6a4cf00d9277fab7b78.

However, it's still failing on the server even when we don't send the fixed fee over:

`PATCH ob/profile/<peer-id>`

```
{
  "moderator": true,
  "moderatorInfo": {
    "fee": {
      "feeType": "PERCENTAGE",
      "percentage": 0.5
    },
    "description": "Describe the scribe.",
    "termsAndConditions": "The terms are germs.",
    "languages": [
      "en-US"
    ]
  }
}
```

Can you please log the bug in the ob-go repo? (I would mention that we're using `PATCH`. I'm not sure if the same behavior also happens on `PUT/POST`)

**Comment by jjeffryes** [comment]
Closed, the server version is at https://github.com/OpenBazaar/openbazaar-go/issues/1881
